qemu/tests/win_virtio_driver_update_test.py

```diff
@@ -39,13 +38,21 @@ def change_virtio_media(cdrom_virtio):
         logging.info("Changing virtio iso image to '%s'" % virtio_iso)
         vm.change_media("drive_virtio", virtio_iso)
 
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
```

Comment:
Do we still need it since you call "session = vm.wait_for_login(timeout=timeout)"?
kind of dup.

Reply:
I think it's okay to leave it here. 
it's verify vm's status.

Reply:
@xiagao I agree with @lijinlijin It only wastes time, not necessary to verify it twice.

Reply:
update.